QML_HUD/main.cpp

```
@@ -0,0 +1,53 @@
+#include "vbar.h"
+
+#include <QGuiApplication>
+#include <QQmlApplicationEngine>
+#include <QQmlContext> 
+#include <QObject>
+#include <QDebug>
+#include <iostream>
+using namespace std;
+
+// Klasa backendu
+class Backend : public QObject
```

cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_PROPERTY is a macro then please configure it.

(unknownMacro)